packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/helpers/getQuestionIds.js

```diff
@@ -0,0 +1,12 @@
+export default function getQuestionIds (
+  choiceId = '',
+  task = {}
+) {
+  if (task.choices[choiceId].noQuestions) {
```

Comment:
```suggestion
  if (task.choices && task.choices[choiceId]?.noQuestions) {
```

Just a bit of safety here too